common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -687,6 +687,129 @@ def test_data_download_a11y(self):
+        When I visit the "Data Download" tab
+            Then I do not see a button to 'List enrolled students' profile information'
+            Examples:
+            | Role          |
+            | instructor    |
+            | staff         |
+
+        """
+        username, __, email, password = self.log_in_as_instructor(
+            global_staff=False,
+            course_access_roles=role
+        )
+        instructor_dashboard_page = self.visit_instructor_dashboard()
+        data_download_section = instructor_dashboard_page.select_data_download()
+
+        self.assertTrue(data_download_section.check_enrolled_student_profile_button_presence)
+        LogoutPage(self.browser).visit()
+        for __ in range(5):
+            learner_username = "test_student_{uuid}".format(uuid=self.unique_id[0:8])
+            learner_email = "{user}@example.com".format(user=learner_username)
+
+            # Enroll test users in the course
+            AutoAuthPage(
+                self.browser,
+                username=learner_username,
+                email=learner_email,
+                course_id=self.course_id
+            ).visit()
+
+        # Login again with staff or instructor
+        AutoAuthPage(
+            self.browser,
+            username=username,
+            email=email,
+            password=password,
+            course_id=self.course_id,
+            staff=False,
+            course_access_roles=role
+        ).visit()
+
+        instructor_dashboard_page = self.visit_instructor_dashboard()
+        instructor_dashboard_page.select_data_download()
+        self.assertFalse(data_download_section.check_enrolled_student_profile_button_presence)
+
+    @ddt.data(['staff'], ['instructor'])
+    def test_view_grading_configuration(self, role):
+        """
+        Scenario: View the grading configuration
+        Given I am "<Role>" for a course
+        When I click "Grading Configuration"
+            Then I see the grading configuration for the course
+            Examples:
+            | Role          |
+            | instructor    |
+            | staff         |
+        """
+        expected = u"""-----------------------------------------------------------------------------
+Course grader:
+<class 'xmodule.graders.WeightedSubsectionsGrader'>
+
+Graded sections:
+  subgrader=<class 'xmodule.graders.AssignmentFormatGrader'>, type=Homework, category=Homework, weight=0.15
+  subgrader=<class 'xmodule.graders.AssignmentFormatGrader'>, type=Lab, category=Lab, weight=0.15
+  subgrader=<class 'xmodule.graders.AssignmentFormatGrader'>, type=Midterm Exam, category=Midterm Exam, weight=0.3
+  subgrader=<class 'xmodule.graders.AssignmentFormatGrader'>, type=Final Exam, category=Final Exam, weight=0.4
+-----------------------------------------------------------------------------
+Listing grading context for course {}
+graded sections:
+[]
+all graded blocks:
+length=0""".format(self.course_id)
```

Comment:
indentation is weird above?

Reply:
yup, it does. But it has to be this way because if you see in the corresponding lettuce test, it has the same indentation as here. The reason is, it fails in the assertion if there are any extra spaces found in the expected string.